### ISSUE TITLE
Training Details

### ISSUE THREAD

**Issue by ForeUP** [tinyvision/SOLIDER-REID#7]
I try to train **Swin-T** with my device, but I can't get similar performance as you uploaded `.pth file` : `91.6/96.1`.
I use `batch_size=64`, train `180 epochs`, other configurations such as learning rate are set by default in your configuration file.

**The results as below:**
```shell
2023-05-02 18:15:32 transreid.train INFO: Validation Results - Epoch: 180
2023-05-02 18:15:32 transreid.train INFO: mAP: 85.1%
2023-05-02 18:15:32 transreid.train INFO: CMC curve, Rank-1  :94.2%
2023-05-02 18:15:32 transreid.train INFO: CMC curve, Rank-5  :98.1%
2023-05-02 18:15:32 transreid.train INFO: CMC curve, Rank-10 :98.7%
```
Could you share more **Training Details**, Thanks a lot!

**Comment by cwhgn** [comment]
Thanks for your interests. For the convenience of your re-implementation,  we upload our training logs into the repo, you can find them in the README.md.

**Comment by ForeUP** [comment]
Thanks a lot!
I think I know the reason why the error occurred. I used the **Swin-small** pretrained model with modified `init_weight` code in #4 .
Thanks for your response.
